prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -5,21 +5,20 @@ postsubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/gcp-guest/osconfig-agent-build:latest
-        imagePullPolicy: Always
+      - image: gcr.io/compute-image-tools-test/osconfig-agent-build:latest
         command:
-        - "/main.sh"
+        - "/run_daisy_workflow.sh"
         volumeMounts:
         - name: compute-image-tools-test-service-account
           mountPath: /etc/compute-image-tools-test-service-account
-          readOnly: true
+          readonly: true
```

Comment:
why remove camelCase?

Reply:
oops... this was not intended. will revert it